tools/codeformat.py

```diff
@@ -168,13 +169,21 @@ def batch(cmd, files, N=200):
 
     # Format C files with uncrustify.
     if format_c:
-        batch(["uncrustify", "-q", "-c", UNCRUSTIFY_CFG, "-lC", "--no-backup"], lang_files(C_EXTS))
+        command = ["uncrustify", "-c", UNCRUSTIFY_CFG, "-lC", "--no-backup"]
+        if not args.v:
+            command.insert(1, "-q")
```

Comment:
probably can just do command.append("-q")? which would be slightly more readable